src/Data/Tree/AVL/RocksDBStore.hs

```diff
@@ -0,0 +1,59 @@
+
+{-# LANGUAGE TypeSynonymInstances #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE MultiParamTypeClasses  #-}
+{-# LANGUAGE FlexibleContexts       #-}
+{-# LANGUAGE ScopedTypeVariables       #-}
+{-# LANGUAGE ExplicitForAll       #-}
+
+module Data.Tree.AVL.RocksDBStore where
```

Comment:
This module is OK, but:
1) After this PR (https://github.com/serokell/lootbox/pull/2) in Lootbox is merged, we should use it instead of raw `rocksdb-haskell-ng`
2) Seems like this module is anyway useful only for demo and testing, because when we'll use AVL+ tree in actual project, we'll use its monad with DB capabilities instead of `RocksDBStore`.

Reply:
This is a proper _implementation_ of the `KVStoreMonad` class. I planned to use it, like, in production.